### ISSUE TITLE
Plugin name not passed

### ISSUE THREAD

**Issue by seldimi** [wphuman/WordPress-Settings-Module#12]
Hey, this is a very good Settings Module.
The only hard part is that we have to rename everything to match the plugin we put it on.

Is there any way to pass $plugin_name than handwriting it ? (eg _Settings_Definition class)

**Comment by tangrufus** [comment]
About renaming: No.
This module is written in 2014 when namespacing a not a thing. It requires you to prefix plugin names to avoid class name conflicts.

## Important!
This module is not maintained anymore. The team `WP Human` has been disbanded.
Use https://github.com/Typisttech/wp-better-settings instead.

Main difference:
- No more hard code `plugin-name`. `WP Better Settings` thinks you should put it under your own namespace to avoid conflict.
- Option are not stored as a single array (to fits in WP REST API)
- Use metaboxes and `postbox.js` properly
- PHP `7.0` or above, PHP 5 doesn't work

Tips: 
`WP Better Settings` is hard to follow at first. Multiple projects with multiple schools of developers... Sorry...
Use factory methods if there is one.

**Comment by tangrufus** [comment]
Deprecated notice added to `readme.md`

**Comment by seldimi** [comment]
Does that mean that this won't be usuably on PHP 7 ? 
As its for small plugin, not big dea

**Comment by tangrufus** [comment]
WP Better Settings only works on php 7.0 or later. 

WordPress Settings Module (this repo) never tested on php7
